### DISCUSSION TITLE
ND remake

### DISCUSSION THREAD

**Discussion by zakiack** [magknight/787aviatorsTracker#592] [Completed things] [answered]
Would love to see a better ND than the current one which is in aircraft. Rn it leaves a lot to desire with as it uses the default X-Plane icons instead of realistic Boeing icons (the star for enroute and active waypoints especially), and overall has a very unpolished feel to it.

Something which looks more like this would be very nice.
![image](https://user-images.githubusercontent.com/76414253/118896063-36578580-b8bc-11eb-95c2-43eb479aaeb7.png)

Thank you for taking this into consideration and great job with v1.7

**Comment by aeroniemi** [comment, marked as answer]
has been completed as part of ND2, currently on cygnus as 1.7.18-alpha
